src/MockNet/Http/Extensions/StringBuilderExtensions.cs

```diff
@@ -25,7 +27,41 @@ public static StringBuilder AppendContent(this StringBuilder sb, string content)
             return sb;
         }
 
+        public static StringBuilder Join(this StringBuilder sb, Func<StringBuilder, StringBuilder> separator, IEnumerable<string> values)
```

Comment:
Nitpick, but can we rename `sb` to `builder`?

Reply:
Updated